onnxruntime/contrib_ops/cuda/bert/tensorrt_fused_multihead_attention/flash_attention/fmha_flash_attention.h

```diff
@@ -269,7 +269,7 @@ class FusedMultiHeadFlashAttentionKernel
     ORT_ENFORCE(headsize <= 0x3FFFFFFF);
     ORT_ENFORCE(qStep <= 0xFFFF);
     ORT_ENFORCE(kvStep <= 0xFFFF);
-    return static_cast<uint64_t>(qStep << 16 | kvStep) << 32 | (headsize << 2) | (interleaved ? 2U : 0U) | (unroll ? 1U : 0U);
+    return static_cast<uint64_t>(qStep << 16 | kvStep) << 32 | (static_cast<uint64_t>(headsize) << 2) | (interleaved ? 2U : 0U) | (unroll ? 1U : 0U);
```

Comment:
Change headsize to a bigger type first to avoid overflow. It is just to suppress a warning. No real effect.